data_collection/gazette/spiders/ce/ce_cedro.py

```diff
@@ -0,0 +1,11 @@
+from datetime import date
+
+from gazette.spiders.base.adiarios_v1 import BaseAdiariosV1Spider
+
+
+class CeCedroSpider(BaseAdiariosV1Spider):
+    TERRITORY_ID = "2303808"
+    name = "ce_cedro"
+    allowed_domains = ["cedro.ce.gov.br"]
+    BASE_URL = "https://www.cedro.ce.gov.br"
+    start_date = date(202, 6, 7)
```

Comment:
```suggestion
    start_date = date(2018, 1, 1)
```
As duas datas iniciais estão fora de ordem e já existem na ordem correta. A data da primeira versão é 01/01/2018.